[KSP Update] RealHeat KSP 1.3 updates

PhineasFreak opened the pull request:
**Change Log:**

* Add a simple GitHub readme.
* Update the version checker.
* Update the assembly version.
* Update the project **.userprefs** file (was pointing to the Deadly Reentry assemblies).

**Notes:**

* The **String** parameter was converted into lowercase **string**.

blowfishpro commented on Source/Checkers.cs:
```diff
@@ -41,6 +41,7 @@ namespace RealHeat
      */
 
     [KSPAddon(KSPAddon.Startup.Instantly, true)]
+
```
Any particular reason for the added newline here?

PhineasFreak replied:
Probably a "mis-newline". Will fix.

blowfishpro commented on Source/Checkers.cs:
```diff
@@ -84,7 +85,8 @@ public static bool IsUnityCompatible()
 
         // Version of the compatibility checker itself.
-        private static int _version = 5;
+
```
and here?

PhineasFreak replied:
Just to separate the var from the function for easier readability. I can supress it if you want.

blowfishpro replied:
Isn't it already separated though?  This newline is between the comment and the field.  It just seemed odd to me.

PhineasFreak replied:
OK, will also fix.

DRVeyl commented:
PhineasFreak, the binary in your RealHeat-RH-KSP-1.3-Binaries branch is v4.3, ie an older build.  Would it make sense to compile & commit here, or post via dropbox?

PhineasFreak commented:
Yes, i am aware of that. I was expecting to make the actual binary after everything settles down and RSS has an official release for KSP 1.3.

PhineasFreak commented:
**Unofficial** RH assembly (for testing purposes only): https://github.com/PhineasFreak/DocBin/blob/master/DocBin/dll_files/RealHeat.dll

eggrobin commented:
@PhineasFreak, this pull request needs to update the expected KSP version in Checkers.cs. Further, it needs some changes to calls to the popup functions, whose signatures changed between 1.3.0 and 1.3.1, and the binary needs to be rebuilt for 1.3.1.

With the above, LGTM.

PhineasFreak commented:
But i updated the version.minor field to "3". The RH checker does not seem to care about the revision number so everything should be OK. Did i miss something else?

Also, forgive my ignorance, but what signatures do you mean? If you mean the new PopupDialog "titleExtra" field then i have updated the code accordingly.

As far as recompiles go, i was hoping that someone else would do it as part of the new release (since the repository itself never included the actual RH .dll assembly, only the code). If adding the .dll file to the repository is not a problem, then i guess that i could go ahead and do so.

NathanKell commented:
I'll rebuild after the merge. Thanks for this @PhineasFreak !! ❤️

eggrobin commented:
@PhineasFreak, looking at my git log on my testing branch, it seems I forgot to actually patch in this pull request before testing, hence the nonsensical comments... :-p

PhineasFreak commented:
@eggrobin Heh, no problem! I had this happen to me before, so i can relate! :smile:
